### PR TITLE
Add more style to the (newsletter) sign up button in devhub

### LINKED ISSUE #12700: The Sign up button from the Connect with us section doesn't respond to mouse hovers
Follow up for #12613 

### Describe the problem and steps to reproduce it:
1. Open DevHub homepage in logged out view - https://addons.allizom.org/en-US/developers/
2. Scroll down to the Connect with us section (blue footer)
3. Hover over the Sign up button form the Extensions Developer Newsletter component

### What happened?
There is no response from the button on mouse hover (mouse pointer doesn't change)

### What did you expect to happen?
The pointer should change when hovering over the button to indicate an interaction

### Anything else we should know?
- I've added a comparison with Extension workshop to illustrate what the expected result should be

[DevHub]

[Extension workshop]

### PR DESCRIPTION
**Pull request by BenTarman**
Fixes #12700

**BEFORE (all states)** 
beforeAllStates

 **AFTER default** 
afterDefault

 **AFTER active** 
afterActive

 **AFTER hover** 
afterHover

### CODE REVIEW

File: static/css/devhub/new-landing/sections/connect.less [hunk 1]
```diff
@@ -109,10 +109,21 @@
 
         button {
             height: 32px;
-            .button(@color-scheme-dark-background, @color-scheme-dark-text);
+            min-width: 5.25rem;
```
[the remark this answers is not in the document]
Comment (eviljeff) [reply static/css/devhub/new-landing/sections/connect.less]: I'm not sure either way, but if we're adding `width: 100%` it needs to be tested with all page widths from 230px to 1200px to check it's not broken the layout elsewhere.

File: static/css/devhub/new-landing/sections/connect.less [hunk 2]
```diff
@@ -109,10 +109,21 @@
...
+            .button(#015fdf, #fefefe);
```
[the remark this answers is not in the document]
Comment (BenTarman) [reply static/css/devhub/new-landing/sections/connect.less]: okay thanks. You are good with these colors or want the original ones?

File: static/css/devhub/new-landing/sections/connect.less [hunk 3]
```diff
@@ -109,10 +109,21 @@
...
             font-size: 14px;
-            border-radius: 4px;
+            border-radius: 0.25rem;
             border-width: 0;
+            cursor: pointer;
+            transition: background-color 0.25s ease-out;
+
+            &:hover {
```
[the remark this answers is not in the document]
Comment (BenTarman) [reply static/css/devhub/new-landing/sections/connect.less]: I think adding background-color and border color as parameters to the mixin (and just default them to what we hardcoded) would be good. I assume that is what your getting at.

**Comment by BenTarman** [comment]
okay I just reverted my changes to basically be what we had before with the mixin since I think that is what was intended on the issue.

Kept the transition though. Let me know if we did want the colors I had in the picture and I can add that as variables (and edit the mixin to take a background-color parameter).

**Comment by willdurand** [comment]
> Kept the transition though. Let me know if we did want the colors I had in the picture and I can add that as variables (and edit the mixin to take a background-color parameter).

Yes, that would be nice I guess. Thanks

**Comment by BenTarman** [comment]
Yeah I can get to that later today

**Comment by BenTarman** [comment]
Okay I used the same blue color we had for other text on page. We do have this same color saved to @color-button-register and @color-button-primary-text. I just assumed that was on purpose.

buttonConnect

**Comment by caitmuenster** [comment]
Thanks so much for the patch, @BenTarman! 🎉 Your contribution has been added to our [recognition wiki](https://wiki.mozilla.org/Add-ons/Contribute/Recognition). 

Would you be interested in creating an account on [mozillians.org](https://mozillians.org/)? I'd be happy to vouch for you!

**Comment by BenTarman** [comment]
Thanks @caitmuenster. yeah here is the created profile to vouch: https://mozillians.org/en-US/u/Eango/.

**Comment by caitmuenster** [comment]
Woohoo! You're vouched. ✨

Welcome onboard! I look forward to seeing you around the project.

**Pull request merged**